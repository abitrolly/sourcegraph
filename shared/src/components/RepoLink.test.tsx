import React from 'react'
import { RepoLink } from './RepoLink'
import { mount } from 'enzyme'

describe('RepoLink', () => {
    test('renders a link when "to" is set', () => {
        expect(mount(<RepoLink repoName="my/repo" to="http://example.com" />)).toMatchSnapshot()
    })

    test('renders a fragment when "to" is null', () => {
        expect(mount(<RepoLink repoName="my/repo" to={null} />)).toMatchSnapshot()
    })
})
