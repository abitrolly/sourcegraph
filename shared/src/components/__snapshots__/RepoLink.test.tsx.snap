// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RepoLink renders a fragment when "to" is null 1`] = `
<RepoLink
  repoName="my/repo"
  to={null}
>
   
  my/
  <strong>
    repo
  </strong>
</RepoLink>
`;

exports[`RepoLink renders a link when "to" is set 1`] = `
<RepoLink
  repoName="my/repo"
  to="http://example.com"
>
  <AnchorLink
    className=""
    to="http://example.com"
  >
    <a
      className=""
      href="http://example.com"
    >
       
      my/
      <strong>
        repo
      </strong>
    </a>
  </AnchorLink>
</RepoLink>
`;
